src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goodVisible, setGoodVisible] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+
+  const sortAlphabet = () => {
+    if (sortField.includes('reverse')) {
+      setGoodVisible([...goodVisible].sort((a, b) => b.localeCompare(a)));
+      setSortField('alphabet reverse');
+    } else {
+      setGoodVisible([...goodVisible].sort((a, b) => a.localeCompare(b)));
+      setSortField('alphabet');
```

Comment:
It'll be a better approach to create a separate state to check if it's reversed to maintain a clear and understandable code structure.
And then you could create one function for sorting and check if it's reverse at the end. You have a lot of repeatable logic now